server/app/controllers/checks_controller.py

```diff
@@ -9,12 +13,64 @@
 from app.utils.scrape import scrape_MOSS_report
 
 from datetime import datetime
+from collections import OrderedDict
 
 
 class ChecksController:
+
```

Comment:
This class has no comments.  There should be method comments saying what each method does, and identifying what the parameters are for.  For each step in the code, there should also be a comment saying what it does (e.g., one before line 39 in this file).